src/main/java/svc/controllers/CourtController.java

```diff
@@ -30,11 +34,23 @@ CourtsDTO GetCourts() {
 	
 	@ResponseBody
 	@RequestMapping(method = RequestMethod.GET, value="/{id}")
-	Court GetCourt(@PathVariable("id") Long id) {
-		if (id == null) {
-			LogSystem.LogEvent("Null id passed to controller");
-		}
-		
-		return courtManager.GetCourtById(id);
+	Court GetCourt(@PathVariable("id") Long id) throws NotFoundException {
+		Court court = courtManager.GetCourtById(id);
+		if (court == null) 

```

Comment:
Best practice is to always use curly braces. Many code quality tools will mark this is a warning because it's easy for it to break later.